docs/api/ipc-main.md

```diff
@@ -90,8 +90,14 @@ Set this to the value to be returned in a synchronous message.
 
 ### `event.sender`
 
-Returns the `webContents` that sent the message, you can call
+Returns the `ipcRenderer` that sent the message, you can call
```

Comment:
I'm not sure this would work as a type. As `ipcRenderer` is a module not a class so it can't return an instance of module.

Any idea how can we indicate it's returning a module and not an instance @zeke